client/components/button/style.scss

```diff
@@ -143,9 +143,13 @@ button {
 	&:focus {
 		border-color: $alert-red;
 	}
-	&:focus {
-		box-shadow: 0 0 0 2px lighten( $alert-red, 20% );
+
+	.accessible-focus & {
+		&:focus {
```

Comment:
@folletto Your method compiles the same, so I updated the code in both places. Thanks!